src/components/modals/WelcomeLoadingModal.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Modal, ProgressBar, Alert } from 'react-bootstrap';
+
+import { FaFrog } from 'react-icons/fa';
+
+const WelcomeLoadingModal = ({
+  isVisible,
+  isNewRoom,
+  authenticating,
+  connecting,
+  connected
+}) => {
+
+  let progress = 0;
+  let statusMessage = (
+    <p>Cranking a bunch of gears.</p>
+  );
+
+  if (authenticating) {
+    progress = 30;
+    statusMessage = <p>Establishing a session in your browser.</p>;
+  } else if (connecting) {
+    progress = 60;
+    statusMessage = <p>Creating a secure connection with LeapChat servers.</p>;
+  } else if (connected) {
+    progress = 95;
+    statusMessage = <p>Connected!</p>;
+  }
+
+  return (
+    <Modal show={isVisible}>
+      <Modal.Header closeButton>
+        <h3>Welcome to LeapChat!</h3>
+      </Modal.Header>
+      <Modal.Body>
+        {isNewRoom && <Alert
+          bsStyle="success">
+          New Room Created!{' '}<FaFrog />
+        </Alert>}
+        {!isNewRoom && <Alert
+          bsStyle="success">
+          Successfully joined room!{' '}<FaFrog />
```

Comment:
The frog is cute but people will think it's a [hat-tip to the far right](https://www.theguardian.com/world/2017/may/08/pepe-the-frog-creator-kills-off-internet-meme-co-opted-by-white-supremacists) :disappointed: . Pick something else at least as cool?

Reply:
Oh yeah, I forgot about Pepe. Yah, I'll nuke it.